src/importV1/index.ts

```diff
@@ -0,0 +1,192 @@
+import * as fs from 'fs';
```

Comment:
Nit: `// TODO: file naming consistency, should be 'src/import-v1/'`

Reply:
Renamed.